http/rest-client/src/test/java/io/quarkus/ts/http/restclient/GzipMaxInputIT.java

```diff
@@ -0,0 +1,97 @@
+package io.quarkus.ts.http.restclient;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.util.zip.GZIPOutputStream;
+
+import jakarta.ws.rs.core.MediaType;
+
+import org.apache.http.HttpStatus;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.services.QuarkusApplication;
+import io.restassured.response.Response;
+
+@QuarkusScenario
+public class GzipMaxInputIT {
+
+    final byte[] gzip_bellow_max_input_1K = new byte[1000];
```

Comment:
We have limit of 1k, but two failing tests of 10M and 100M. Do we really need both? They are both way over the limit. 

If we wnat to check borders, than something more reasonable would be 9 kb test (works) 10 kb test(fails) and 1/10/100 Mb test (fails in the same way as 10 kb test. Can it be done?

Sizes of zero and sending valid and invalid content is nice to have, but not required.

Reply:
Yes, I agree with you, I will tweak the tests.